test/validation/api/pktio/pktio.c

```diff
@@ -1196,7 +1196,8 @@ static void test_recv_tmo(recv_tmo_mode_e mode)
 	CU_ASSERT_FATAL(ret == test_pkt_count);
 
 	ret = recv_packets_tmo(pktio_rx, &pkt_tbl[0], &pkt_seq[0], 1, mode,
-			       odp_pktin_wait_time(UINT64_MAX), 0, 0);
+			       odp_pktin_wait_time(10 * ODP_TIME_SEC_IN_NS),
```

Comment:
Looks like a reasonable change, but not clear what this has to do with Ubuntu 20.04.

Reply:
I caught this problem while testing with Ubuntu 20.04. On Focal this call never returns with pcap pktio due to problems with a newer libpcap version. That's why I've not yet enabled running the validation tests with Ubuntu 20.04.

I'll try to fix the pcap pktio on another PR.